include/psf_constants.h

```diff
@@ -19,6 +19,8 @@ namespace psf
 
     constexpr char arch_string[] = "32";
     constexpr wchar_t warch_string[] = L"32";
+
+    constexpr wchar_t current_version[] = L"1.0.221230.1";

```

Comment:
Big possibility for this number to not be updated during a new release.  Connect with the Windows App SDK folks to figure out how they managed this.

My suggestion is to change this number to a constant local copy.  Something like 0.0.05.  Then, during the official build, replace this value with the correct version.